android/src/main/java/com/github/wuxudong/rncharts/charts/CombinedChartManager.java

```diff
@@ -21,6 +22,8 @@ public String getName() {
 
     @Override
     protected CombinedChart createViewInstance(ThemedReactContext reactContext) {
+        mContext = reactContext;
```

Comment:
can the react context change, and if so, is it guaranteed this class would be recreated?

Reply:
`context` should already be stored in `chart` instance, one thought is to access `context` from `chart` whenever needed. since `getDataExtract` would be accessed with `chart` (to setData presumably?), can do something like 
```
getDataExtract(chart.getContext())
```

Reply:
done, good call!